### PR TITLE
add ssl secure redirect - fix #38

### LINKED ISSUE #38: Remove template debug
I would set `template_debug` = `debug` settings

### PR DESCRIPTION
**Pull request by hassanabidpk**
Add Secure SSL Redirect  Reference : https://docs.djangoproject.com/en/1.8/topics/security/#ssl-https
Fix #38

### CODE REVIEW

File: code4every1site/settings.py [hunk 1]
```diff
@@ -21,12 +21,13 @@
 # See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
 
 DEBUG = os.getenv('DJANGO_DEBUG') != 'FALSE'
```
Comment (jinpark) [inline code4every1site/settings.py]: I know this isint part of yoru PR, but what the heck does this mean?

Comment (hassanabidpk) [inline code4every1site/settings.py]: haha This I got from Django girls :D https://github.com/DjangoGirls/djangogirls/blob/master/djangogirls/settings.py These are stored in WSGI in python anywhere- kind of cool

Comment (jinpark) [inline code4every1site/settings.py]: Yea, I see what this means now! Took me a second to understand why its done this way

**Comment by jinpark** [comment]
I'm not sure how to test this. @hassanabidpk how do you test this locally?

**Comment by hassanabidpk** [comment]
I am not sure about it in debug mode. But I will give it a try.

**Comment by hassanabidpk** [comment]
I mean in development

**Comment by jinpark** [comment]
Ok, I tried to test this with `stunnel` but I got lazy and its probably easier to get this out on prod and test it out. we can revert  if it breaks everythng

**Pull request merged**